icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
Still disagree on WxH. Our maintenance burden will increase if we have to make new templates and diverging settings files for every variation in image size from project to project. I'm out of time to discuss it so am going to press ahead with omitting it. I think easy_thumbnails may need to change how it resolves aliases before we're both happy.